.travis.yml

```diff
@@ -46,4 +46,5 @@ after_success: |
   echo "documented"
 env:
   global:
-    secure: DzjCwZae6k/GyYLJ5x3wgDIjO8SjzkAq3eI8RvRT4aqSfVai2x3y4bPk5QoX+lt6TLaJxmoc1Vq6HZJbNlKokpGAvyzwJ0qmCBWYCUcRyTUyAVmH8VEY/wnBF0k8lYiQmfxOeGhiQb8nHZSpIIhItfIUirKMXyCW3ArkVV5HzQa53T1mO3unY9AEnDgyHfmwke0QM6MYdHAhNjfg45dvckU6Euu2oAXZ0tQpf0+uDPpU78Mg3dJ523/REuhKgRjreHt3S99R8rmnlKY9ZQeK0Zg4ZE4e+xxeC0H67CIhLWZiIwqZxauA7zB11UXI7GZ6fZoPsvZN3U3TlJwaAU/RHzGBczZVxGqgujwsbrne2WRYFtnFirs8D15uRp4ix4UIyKaUeqy5rUoZcLAzKIWDNLTJ5bQvyhDN641eF1auPURr0z7g+tQc8uEEKmuJXR+PplHPzvGOTz54dQa603XeEoERjL/59XqcbJFCoW3FqIxlLMosBOyQBfE1eGpkisFUdZ6YYODEFFOggF1JWF5psP9DUJ9CX852EdPpgIzojvmo9KqeeBiHw/I02rTRa6QgsIT0uea5HlV6eNo0le9AeT3fMYfVU/wzDJxH/2mt42SCvLSbbXl9Nam1KUtEvjc5cGhD4vjnX6G2hAiY/xPXuvmlZ8btwlaykeI//wdvq7o=
```

Comment:
Why was this needed?

Reply:
Going forward with your idea of a fallback rather than runtime error it won't be.  I will remove the change.